dev/live_views/main.ex

```diff
@@ -20,13 +20,13 @@ defmodule LiveDebuggerDev.LiveViews.Main do
     <.box title="Main [LiveView]" color="blue">
       <div class="flex flex-col gap-2">
         <div class="flex items-center gap-2">
-          <.button phx-click="increment" color="blue">
+          <.button id="increment" phx-click="increment" color="blue">
```

Comment:
I'd make this IDs a bit more descriptive, to make also sure that there won't be any repetitions in the future
```suggestion
          <.button id="increment-button" phx-click="increment" color="blue">
```